frontend/components/reviews/functions.ts

```diff
@@ -179,6 +200,33 @@ export const getCityOptions = (
 	return []
 }
 
+export const getZipOptions = (
+	data: Array<Review> | undefined,
+): Array<Options> => {
+	if (!data) return []
+	if (data.length) {
+		const allZipOptions = data.map((review, id) => {
+			const zip = review.zip.toLowerCase().trim()
+			return {
+				id: id + 1,
+				name: zip.split(' ').map(capitalize).join(' '),
+				value: review.zip.toLowerCase().trim(),
+			}
+		})
+
+		const zipOptions = removeDuplicates(allZipOptions, 'value')
```

Comment:
Remove Duplicates will need to be fixed to avoid the errors of different styles of entering postal codes (H0H 0H0 vs H0H0H0)

Reply:
Thanks Kellen. I made the additional changes. I am having a bit of trouble having the reviews come through when I host the site locally, so I can't validate very well. Docker is telling me:

2023-04-18 22:16:11 frontend  | API resolved without sending a response for /api/get-reviews, this may result in stalled requests.

Reply:
Okay, I will have to look into that issue. 

Did you setup the environment following the ReadMe?

Reply:
I merged the changes into this request. Apologies. 

I created the .env file per the instructions using the example. Set ENVIRONMENT to development (was already set to that). I am able to make the docker pull, build, and then start.